### PR TITLE
IFG BA registered users

### PR DESCRIPTION
**Pull request by anil469**
If an IFG user registers to BA website then the email of that user has to be unsubscribed from IFG email campaigns.

### CODE REVIEW

File: dbt/brandalley-dbt/models/marketing/emarsys/emarsys_ifg_ba_customers.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    email as ifg_email,
+    2 as ifg_optin
+  from {{ source('emarsys_ifg', 'ifg_*') }}
+  where _TABLE_SUFFIX like '%_transformed'
+)
+select
+    *
+from ba_customers ba
+inner join ifg_customers ifg
+on ba.email=ifg.ifg_email and ba.optin=1
```
Comment (nicklupu) [inline dbt/brandalley-dbt/models/marketing/emarsys/emarsys_ifg_ba_customers.sql:30]: only thing was should lower both emails to make sure this join is case insensitive on email

Comment (anil469) [reply dbt/brandalley-dbt/models/marketing/emarsys/emarsys_ifg_ba_customers.sql]: Resolved it.

**Review by nicklupu — APPROVED** [review]
should there be a unique key on the config of customer id? other than that all looks good to me!

**Comment by anil469** [comment]
> should there be a unique key on the config of customer id? other than that all looks good to me!

customer_id can have duplicates in this case because we have multiple IFG brands. if same email id exists in multiple IFG brands then there is a chance of duplicates in the table and that is fine.

**Pull request merged**